classes/class-wc-connect-shipping-label.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Shipping_Label' ) ) {
+
+	class WC_Connect_Shipping_Label {
+
+		private $settings_store;
+
+		public function __construct( WC_Connect_Service_Settings_Store $settings_store ) {
+			$this->settings_store = $settings_store;
+		}
+
+		protected function get_form_data( WC_Order $order ) {
+			$formData = array();
+			$contents = array();
+
+			foreach( $order->get_items() as $item ) {
```

Comment:
We may (or may not) already have rates and box packing solutions (e.g. if the customer selected a USPS rate during checkout). Please add a TODO and an issue